modules/scanning-images-in-a-remote-cluster.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * cli/image-scanning-by-using-the-roxctl-cli.adoc
+
+:_content-type: PROCEDURE
+[id="scanning-images-in-a-remote-cluster_{context}"]
+= Scanning images in a remote cluster
+
+.Prerequisites
+
+* You have created an image using a Docker file and added it to the {ocp} integrated registry.
+
+* Verify that you have a version of `roxctl` CLI with the `--cluster` flag:
++
+[source,terminal]
+----
+$ roxctl image scan -h
+----
++
+.Example output
++
+[source,text,subs=attributes+]
+----
+[...]
+
+Flags:
+      --cluster string       cluster name or ID to delegate image scan to
```

Comment:
Similarly, would recommend if keeping this section to show more of the parameters / general usage of `roxctl image scan`